README.md

```diff
@@ -0,0 +1,5 @@
+# Makeroid Official Website
```

Comment:
If you could add a little bit more information such us screenshots or what we do it would be 👌

Reply:
OK, I'm going to try my best.